packages/wrangler/src/pages/download-config.ts

```diff
@@ -51,6 +52,12 @@ interface PagesDeploymentConfig extends DeploymentConfig {
 		}
 	>;
 	ai_bindings: Record<string, Record<string, never>>;
+	limits?: {
+		cpu_ms: number;
+	};
+	placement?: {
+		mode: "smart";
```

Comment:
shouldn't this be `{ mode: "off" | "smart" }`?

Reply:
No—to turn smart placement off this field is omitted entirely